### PR TITLE
add questions to faq, update links

### PR DESCRIPTION
**Pull request by karissa**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -10,6 +10,18 @@ Code for Science and Society (CSS), a US based 501(c)(3) not for profit organiza
 
 Dat uses hyperdrive and a variety of other modules. Hyperdrive and Dat are compatible with each other but hyperdrive is able to make lower-level decisions. Dat presents a user-friendly interface and ecosystem for scientists, researchers, and data analysts.
 
+# How do dat peers discover one another on the Internet?
+
+Dat is very flexible. It currently uses 3 methods, and you can implement your own.
+
+ 1) Multicast udp, which lets it work over the local network (LAN), 
+ 2) A distributed hash table, which lets it work without a central server, and 
+ 3) some centralized signaling servers that run a modified version of DNS.
```
Comment (joehand) [inline docs/faq.md]: Maybe note that we run a centralized signaling server that we use as a default in our clients. But others are free to use run the same software and overwrite the defaults in other applications.

Comment (joehand) [inline docs/faq.md]: Added commit for o/

**Comment by joehand** [comment]
Thanks @karissa for getting docs update rolling again! Have a few questions on how we want to organize our faq. 

1. Should we put questions about a specific application (e.g. `Dat on the CLI isn't connecting, how do I debug`) on this page? I think they'd be better on the readme or app-specific doc page.
2. We have a weird mix of technical and non-technical questions. Are folks 👍  to grouping them under **General Dat Info & Usage** and **Under the Dat Hood** or something similar?
3. We have a few more [faq questions in this issue](https://github.com/datproject/docs/issues/14), especially the common ones privacy/history/multiwriter, to add. I can add placeholders (unless we want this merged soon) and start filling them out.

**Comment by joehand** [comment]
Collected [all outstanding questions](https://github.com/datproject/docs/issues/14#issuecomment-272511899) in that issue.

**Comment by karissa** [comment]
Yeah, it'd be helpful to have more parsing out.

The 'dat on the CLI isn't connecting' should probably be in 'troubleshooting' in the readme.

It'd help to have https://github.com/freeman-lab/minidocs/issues/63 to separate 'FAQ> general' and 'FAQ> under the hood'

File: docs/faq.md (hunk 1 above)
Comment (karissa) [inline docs/faq.md]: Yeah makes sense

**Comment by karissa** [comment]
going to merge this now, lets open another PR

**Pull request merged**